devices/xiaomi.js

```diff
@@ -1481,7 +1481,7 @@ module.exports = [
         ota: ota.zigbeeOTA,
     },
     {
-        zigbeeModel: ['lumi.sensor_cube', 'lumi.sensor_cube.aqgl01'],
+        zigbeeModel: ['lumi.sensor_cube', 'lumi.sensor_cube.aqgl01', 'lumi.remote.cagl02'],
```

Comment:
It seems this device has a different model: `MFCZQ12LM`, can you duplicate it this definition and also update the description?